README.md

```diff
@@ -152,3 +152,54 @@ This is the template PVC. No namespace is supplied since the PVC is excpected to
 1. Monitor the importer pod:
 
         $ kubectl logs <unique-name-of-importer pod>  # shown in controller log above
+
+### Security Configurations
+
+#### RBAC Roles
+
+CDI needs certain permissions to be able to execute properly, primarily the `cluster-admin` role should be applied to the service account being used through the [Kubernetes RBAC model](https://kubernetes.io/docs/admin/authorization/rbac/).  For example, if CDI is running in a namespace called `golden-images`, and the `default` service account is being used, then the following RBAC should be applied:
+
+```
+  $ kubectl create clusterrolebinding <binding name> --clusterrole=cluster-admin  --serviceaccount=<namespace>:default
+
+  i.e.
+  $ kubectl create clusterrolebinding c-golden-images-default --clusterrole=cluster-admin  --serviceaccount=golden-images:default
+
+```
+
+> NOTE: This gives full cluster-admin access to this binding
```

Comment:
Has there been any work done to determine the minimal privileges/role needed for the controller to be able to list pvcs in its own namespace?

Reply:
There has been some investigation, but think that will need to be fine tuned some more, but initially we wanted the doc to reflect our ansible role, as it moves forward, we will update the doc as well